### PR TITLE
Process uncaught errors as Airbyte error traces + remove duplicate tests

### PR DESCRIPTION
**Pull request by cjwooo**
## Description

Process uncaught errors as Airbyte error traces + remove duplicate tests

## Type of change
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change

## Related issues

> Fix [#462](https://github.com/faros-ai/airbyte-connectors/issues/462) 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: .github/workflows/skip-test-sources.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+agileaccelerator-source
```
Comment (tovbinm) [inline .github/workflows/skip-test-sources.txt:1]: we dont have acceptance tests for all these sources?

File: faros-airbyte-cdk/src/protocol.ts [hunk 2]
```diff
@@ -181,6 +183,24 @@ export class AirbyteRecord implements AirbyteMessage {
   }
 }
 
+interface AirbyteErrorTrace {
+  type: 'ERROR';
+  emitted_at: number;
+  error: any;
+}
+
+export class AirbyteTraceMessage implements AirbyteMessage {
```
Comment (tovbinm) [inline faros-airbyte-cdk/src/protocol.ts:192]: rename to `AirbyteErrorTraceMessage`?

File: destinations/airbyte-faros-destination/test/converters/gitlab-ci.test.ts [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
+import {AirbyteLog, AirbyteLogLevel} from 'faros-airbyte-cdk';
 import _ from 'lodash';
 import {getLocal} from 'mockttp';
-import os from 'os';
 import pino from 'pino';
 
-import {InvalidRecordStrategy} from '../../src/destination';
 import {CLI, read} from '../cli';
 import {initMockttp, readTestResourceFile, tempConfig} from '../testing-tools';
-import {gitlabCiAllStreamsLog, gitlabCiLog, gitlabCiPGRawLog} from './data';
```
Comment (tovbinm) [inline destinations/airbyte-faros-destination/test/converters/gitlab-ci.test.ts:10]: `gitlabCiLog, gitlabCiPGRawLog` - remove all the unused resource files then (same for other sources)

**Review by tovbinm — COMMENTED** [review]
see comments + update the PR title & description to reflect the changes

File: .github/workflows/skip-test-sources.txt (hunk 1 above)
Comment (cjwooo) [reply .github/workflows/skip-test-sources.txt:1]: Some of these have expired credentials so I'm disabling them temporarily.

File: faros-airbyte-cdk/src/protocol.ts [hunk 4]
```diff
@@ -181,6 +183,24 @@ export class AirbyteRecord implements AirbyteMessage {
...
+  readonly type: AirbyteMessageType = AirbyteMessageType.TRACE;
+  private readonly trace: AirbyteErrorTrace;
+  constructor(error: any) {
+    this.trace = {
+      type: 'ERROR',
```
[the remark this answers is not in the document]
Comment (cjwooo) [reply faros-airbyte-cdk/src/protocol.ts:197]: This is what the test expects.

**Review by tovbinm — APPROVED** [review]

**Pull request merged**